config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml

```diff
@@ -147,3 +147,64 @@ presubmits:
             memory: "6Gi"
     annotations:
       testgrid-create-test-group: 'true'
+  - name: pull-kubernetes-node-kubelet-serial-cpu-manager
+    skip_branches:
+    - release-\d+\.\d+  # per-release image
+    annotations:
+      testgrid-dashboards: wg-resource-management
```

Comment:
This is the same testgrid that is used for CPUManager CI. New tab for both the CPUManager PR job, and TopologyManager PR job. Please take a look, and let me know looks good or if you have another preference. Thanks.